lib/active_merchant/billing/gateways/ixopay.rb

```diff
@@ -3,17 +3,22 @@
 module ActiveMerchant #:nodoc:
   module Billing #:nodoc:
     class IxopayGateway < Gateway
-      self.test_url = 'https://secure.ixopay.com/transaction'
-      self.live_url = 'https://secure.ixopay.com/transaction'
+      self.test_url = 'https://secure.cardflo.io/transaction'
+      self.live_url = 'https://secure.cardflo.io/transaction'
+      TRANSACTION_WITH_CARD_URL = 'https://secure.cardflo.io/Schema/V2/TransactionWithCard'
```

Comment:
instead of Constant, I told you to define like : `self.transaction_with_card_url`

Reply:
we don't have this class attribute in the parent class, 
is it worth adding it instead of constant?